backend/question.py

```diff
@@ -6,3 +6,73 @@
 @question_page.route('/question/ping', methods=['GET'])
 def question_ping():
     return "Pong", 200
+
+# allow user to post a question and word account not more than 200 words
+@question_page.route('/question/add', methods=['POST'])
+def question_add():
+    data = request.get_json()
+    user_id = data['user_id']
+    question = data['question']
+    answer = data['answer']
+    if len(question) > 200:
+        return make_response(jsonify({"error": "Question is too long"})), 400
+    if len(answer) > 200:
+        return make_response(jsonify({"error": "Answer is too long"})), 400
+    con = sqlite3.connect(DATABASE_NAME)
+    cur = con.cursor()
+    sql = "insert into questions values (?, ?, ?, ?)"
```

Comment:
question table is like this, in app.py

question and answer wont be in the same request. Answer will be in the comments table 

"CREATE TABLE IF NOT EXISTS questions ( \
        id integer primary key autoincrement, \
        title text, \
        content text, \
        timeCreated numeric, \
        timeUpdated numeric, \
        author numeric, \
        replyIds text, \
        thumbUpBy text, \
        isDeleted NOT NULL DEFAULT '0') \
        ")